File: CPP/Assignment/day9/Employee/Employee.cpp
#include<iostream>
#include"Employee.h"
using namespace std;
Employee::Employee(){
    cout<<"Employee::Employee";
}

Employee::Employee(int empId,string name,double salary):empId(empId),name(name),salary(salary){
    cout<<"\n Employee::Employee(int string double)"<<endl;
}

void Employee::Accept(){
    cout<<"Enter Employee ID:->";
    cin>>empId;
    cout<<"Enter employee Name:->";
    cin>>name;
    cout<<"Enter Employee Salary:->";
    cin>>salary;
}
void Employee::Display(){
    cout<<"[ "<<"employee ID: "<<empId<<" employee name "<<name<<" employee slary: "<<salary<<" ]"<<endl;
}
 

File: OOPWJAVA/Assignment/Day4/EmployeeManagementSystem/src/com/cdac/acts/employee.java
package com.cdac.acts;

public class employee {
	private int empId;
	private String name;
	private double salary;
	private static int baseEmpId = 1001;
	
	public employee(){
		super();
		this.empId = baseEmpId++;
		this.name = "";
		this.salary = 0;
	}
	
	public employee(String name, double salary) {
		super();
		this.empId = baseEmpId;
		this.name = name;
		this.salary = salary;
	}
	
	public int getEmpId() {
		return empId;
	}
	
	public double getSalary() {
		return salary;
	}

	@Override
	public String toString() {
		return "employee [empId=" + empId + ", name=" + name + ", salary=" + salary + "]";
	}
	
	
	
}
